sdk/core/Azure.Core/CHANGELOG.md

```diff
@@ -0,0 +1,23 @@
+# Release History
+
+## 1.0.0
+
+- GA release of Azure.Core package.
```

Comment:
If there were no changes significant enough to detail, would it make sense to just note that?   Something like  "Updating versioning and packaging for general availability; no significant changes from preview.9 were made."   ?